HTML/HTML.sublime-syntax

```diff
@@ -269,7 +269,7 @@ contexts:
 
   doctype-meta:
     - meta_scope: meta.tag.sgml.doctype.html
-    - match: '>'
```

Comment:
Personally, I prefer the quoted version instead of a backslash escape here because `>` as syntax is interpreted by YAML, not by the regular expression engine, where the escape is simply ignored (but not in boost, for example).

Since we need to escape > for YAML, I say we should also use YAML's means for doing so through a quoted scalar. The same applies to other syntax punctuation like colons and quotation marks.